src/lib/rc/common_rc.h

```diff
@@ -27,3 +27,7 @@ extern rc_decode_buf_t rc_decode_buf;
 
 uint8_t crc8_dvb_s2(uint8_t crc, uint8_t a);
 uint8_t crc8_dvb_s2_buf(uint8_t *buf, int len);
+
+#define RC_INPUT_RSSI_MAX		100		//should be same as  input_rc_s::RC_RSSI_MAX
```

Comment:
Why can't we use these from input_rc_s directly?

Reply:
@dagar Thanks
I must say I am not sure why, the original pr was almost a year ago.
so fixed now.
About the defines on the px4io I used the none cpp defines